bitnami/Dockerfile

```diff
@@ -153,5 +153,6 @@ RUN set -ex \
         /var/lib/apt/lists/* \
         /tmp/*               \
         /var/tmp/*
+RUN sed -r -i "s/[#]*\s*(shared_preload_libraries)\s*=\s*'(.*)'/\1 = 'timescaledb,\2'/;s/,'/'/" /opt/bitnami/postgresql/share/postgresql.conf.sample
```

Comment:
This should be unnecessary since `timescaledb-tune` will add it to the shared_preload_libraries -- can you see if there is a reason that isn't happening?

Reply:
I'll look into it and modify the PR if I find something actionable.

Reply:
The startup process crashes on something before it has a chance to run `timescaledb-tune`.

Reply:
Right. The first init script already tries to do `CREATE EXTENSION timescaledb` but that fails because the library hasn't been preloaded.

Reply:
So it works if I either:

1. Modify `postgresql.conf.sample` in the Dockerfile (like the main image does)
1. Skip the creation of the timescale extension in the `template` and `postgres` databases

Reply:
Ah, good catch then. Thank you!